### PR TITLE
io.github.amit9838.mousam

### PR DESCRIPTION
**Pull request by amit9838**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: io.github.amit9838.weather.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	"data/dependencies/python3-requests.json",
```
Comment (hfiguiere) [inline io.github.amit9838.weather.json:26]: this file is missing.

Comment (amit9838) [reply io.github.amit9838.weather.json:26]: added thanks

**Comment by amit9838** [comment]
bot, build io.github.amit9838.mousam

File: io.github.amit9838.weather.json
Comment (hfiguiere) [inline io.github.amit9838.weather.json:1]: filename doesn't match app-id

**Comment by amit9838** [comment]
bot, build io.github.amit9838.mousam

**Comment by amit9838** [comment]
exactly, the main goal was to change the app_id

**Comment by amit9838** [comment]
i'm plannig to end the life cycle of io.github.amit9838.weather

**Comment by hfiguiere** [comment]
this was never mentioned in the submission

**Comment by amit9838** [comment]
> this was never mentioned in the submission

https://github.com/flathub/flathub/pull/4918/commits/6caa035688188bb480e35985944b4c707b949fd1

I thought that this as a valid solution, found this in docs.

**Pull request closed without merging**